client/js/jsonedit.js

```diff
@@ -1742,7 +1746,8 @@ function jeLoggingRoutineOperationEnd(problems, variables, collections, skipped)
 
 function jeLoggingRoutineOperationSummary(definition, result) {
   jeRoutineResult = `<span class="jeLogSummary">${html(definition)}</span>
-     ${result ? '=&gt;' : ''} <span class="jeLogResult">${html(result || '')}</span>`;
+     ${result ? '=&gt;' : ''} <span class="jeLogResult">${html(result || '')}</span> <span class="jeLogTime">${+new Date() - jeLoggingStart}ms</span>
+`;
```

Comment:
I'm pretty sure that the time printed after an operation is the total elapsed time since execution of the routine started. If `jeLoggingRoutineOperationEnd` set `jeLoggingStart` to the current time, this would (I think) produce incremental times for each operation.

Reply:
Yes. That was semi-intended. Both ways have pros and cons but changing it would require `jeLoggingStart` to be a stack so it can measure sub-operation times individually while still showing the sum (for example `CALL took 100ms; SELECT inside CALL took 30ms`).

Your comment shows that it doesn't match expectations as is though.

Reply:
See commit 37b277a.